Source/SwiftCarouselDelegate.swift

```diff
@@ -30,10 +30,11 @@
      
      - parameter item:  Item that is selected. You can style it as you want.
      - parameter index: Index of selected item that you can use with your data source.
+     - parameter tapped: Indicate that the item has been tapped
```

Comment:
I would add to this that if it returns true it means that it was tapped before the selection, and if it is false it was scrolled.